File: jvm/src/test/kotlin/com/jetbrains/signatureverifier/tests/ConsoleLogger.kt
package com.jetbrains.signatureverifier.tests

import com.jetbrains.signatureverifier.ILogger
import org.jetbrains.annotations.NotNull

class ConsoleLogger : ILogger {
  companion object {
    @JvmStatic
    val Instance = ConsoleLogger()
  }

  override fun Info(@NotNull str: String) = println("INFO: $str")
  override fun Warning(@NotNull str: String) = println("WARNING: $str")
  override fun Error(@NotNull str: String) = println("ERROR: $str")
  override fun Trace(@NotNull str: String) = println("TRACE: $str")
}

File: net/JetBrains.Serialization/tests/ConsoleLogger.cs
using System;
using JetBrains.SignatureVerifier;

namespace JetBrains.Serialization.Tests
{
  internal sealed class ConsoleLogger : ILogger
  {
    public static readonly ILogger Instance = new ConsoleLogger();

    private ConsoleLogger()
    {
    }

    void ILogger.Info(string str) => Console.WriteLine($"INFO: {str}");
    void ILogger.Warning(string str) => Console.Error.WriteLine($"WARNING: {str}");
    void ILogger.Error(string str) => Console.Error.WriteLine($"ERROR: {str}");
    void ILogger.Trace(string str) => Console.Error.WriteLine($"TRACE: {str}");
  }
}